include/votca/tools/structureparameters.h

```diff
@@ -105,7 +105,7 @@ class StructureParameters {
   T get(const StructureParameter parameter) const;
 
  private:
-  std::unordered_map<StructureParameter, boost::any> parameters;
+  std::unordered_map<StructureParameter, boost::any, std::hash<int>> parameters;
```

Comment:
Why is that needed?

Reply:
It turns out tools was also broken on ubuntu 16 in an unrelated issue. This fixes that.

Reply:
Ok, Ubuntu 16 is pretty old, correct? Was the boost bump related?

Reply:
Um no, that was for the floating point comparison issue in akimaspline. I wanted to use the boost epsilon which is only available starting in boost 1.63

Reply:
Yes, 16 has been out for a while.